experimental/packages/otlp-exporter-base/src/transport/fetch-transport.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { IExporterTransport } from '../exporter-transport';
+import { ExportResponse } from '../export-response';
+import { diag } from '@opentelemetry/api';
+import {
+  isExportRetryable,
+  parseRetryAfterToMills,
+} from '../is-export-retryable';
+
+export interface FetchTransportParameters {
+  url: string;
+  headers: () => Record<string, string>;
+}
+
+class FetchTransport implements IExporterTransport {
+  constructor(private _parameters: FetchTransportParameters) {}
+
+  async send(data: Uint8Array, timeoutMillis: number): Promise<ExportResponse> {
+    const abortController = new AbortController();
+    const timeout = setTimeout(() => abortController.abort(), timeoutMillis);
+    try {
+      const isBrowserEnvironment = !!globalThis.location;
+      const url = new URL(this._parameters.url);
+      const response = await fetch(url.href, {
+        method: 'POST',
+        headers: this._parameters.headers(),
+        body: data,
+        signal: abortController.signal,
+        keepalive: isBrowserEnvironment,
+        mode: isBrowserEnvironment
+          ? globalThis.location?.origin === url.origin
+            ? 'same-origin'
+            : 'cors'
```

Comment:
Our setup has a collector running on `oltp.company.com` and applications on `app1.company.com` and `app2.company.com`. The collector `at  oltp.company.com` requires an auth cookie to be sent with each request.

Is it possible to be add the `credentials: 'include'` option here, so we can use the default `fetch` transport implementation instead of our own?